### ISSUE TITLE
Trakt Fails to Start

### ISSUE THREAD

**Issue by scottia** [Razzeee/script.trakt#128]
Have a question, don't think its so much an issue.  I recently swapped boxes (3 days ago) and get the following error now using the latest Trakt master updated this am.  My old box, no matter how many times I revert back works perfectly fine with the latest master.

For the life of me I cannot figure out what its trying to tell me.  XBMC is up, running all other addon's and video with no errors.  I only get this now from trakt.

16:29:03 T:4120  NOTICE: -->Python Interpreter Initialized<--
16:29:03 T:4120   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'sqlite3.OperationalError'>
                                            Error Contents: unable to open database file
                                            Traceback (most recent call last):
                                              File "C:\XBMC\addons\script.trakt\script.py", line 239, in <module>
                                                Main()
                                              File "C:\XBMC\addons\script.trakt\script.py", line 233, in Main
                                                q = sqliteQueue.SqliteQueue()
                                              File "C:\XBMC\addons\script.trakt\sqliteQueue.py", line 61, in __init__
                                                with self._get_conn() as conn:
                                              File "C:\XBMC\addons\script.trakt\sqliteQueue.py", line 77, in _get_conn
                                                self._connection_cache[id] = sqlite3.Connection(self.path, timeout=60)
                                            OperationalError: unable to open database file
                                            -->End of Python script error report<--

Any information, much appreciated.

**Comment by scottia** [comment]
I forgot to add my Db is shared across all devices and all devices have the same issue, except the box I replaced.  Yes I did a full restore from OLD to NEW, so I would expect any local databases from the old device to be the same on the new since both use the same shared database.

I was also reading that the following advancedsettings could have an affect on addon's attempting to read these database values but still its not a problem with the older device.

<videolibrary>
	<importwatchedstate>true</importwatchedstate>
	<importresumepoint>true</importresumepoint>
</videolibrary>
